pkg/apiserver/apiserver.go

```diff
@@ -127,23 +128,53 @@ func runE(flags *flags, log logr.Logger) error {
 		return fmt.Errorf("creating client: %w", err)
 	}
 
+	// Set up cache for account
+	accountCache, err := cache.New(cfg, cache.Options{
+		Scheme: scheme,
+		Mapper: mapper,
+	})
+	if err != nil {
+		return fmt.Errorf("creating cache for account: %w", err)
+	}
+	if err := v1.RegisterAccountUsernameFieldIndex(accountCache); err != nil {
+		return fmt.Errorf("fail to register field index for Account Username: %w", err)
+	}
+	accountClient := &client.DelegatingClient{
+		Reader:       accountCache,
+		Writer:       c,
+		StatusClient: c,
+	}
+	ctx := context.Background()
```

Comment:
You could make this cancelable context, and kill it when the `ctrl.SetupSignalHandlers()` terminate.

Reply:
What is the benefit of this approach? Why don't we just pass the same ctx also in `cache.Start`?

Reply:
I see your point, fixed